.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,18 +1,12 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-title: "[Feature Request] "
-labels: enhancement
+title: ''
+labels: ''
```

Comment:
Why not label these?

Reply:
Why indeed, I didn't mean to remove the label/title.

Reply:
Fixed